tests/sentry/deletions/test_team.py

```diff
@@ -14,11 +15,11 @@ def test_simple(self):
         assert project1.teams.first() == team
         assert project2.teams.first() == team
 
-        deletion = ScheduledDeletion.schedule(team, days=0)
+        deletion = RegionScheduledDeletion.schedule(team, days=0)
         deletion.update(in_progress=True)
 
         with self.tasks():
-            run_deletion(deletion.id)
+            run_deletion(deletion.id, silo_mode=str(SiloMode.REGION))
```

Comment:
This feels weird but seems to be the correct approach from what I can tell

Reply:
Correct.